translations/base-tr.yaml

```diff
@@ -641,10 +647,12 @@ storyRewards:
             gibi çalışır, fakat <strong>iki şekli birden</strong> boyayarak iki
             boya yerine sadece bir boya harcar!
     reward_storage:
-        title: Depo
-        desc: <strong>Depoyu</strong> açtınız! - Gönderdiğin eşyaları içinde biriktirir!<br><br>
-            Öncelikli olarak sol tarafından eşyaları gönderir eğer sol taraf tıkanırsa sağ taraftan gödermeye başlar.
-            Bunu <strong>fazlalık eşyaları</strong> değerlendirmek için kullanabilirsin!
+
+        title: Depo Sağlayıcı
+        desc: <strong>depolamayı</strong>açtın - Bu eşyaları depolamayı sağlar
+            <br><br> sol çıkışa öncelik verir.
+            buyüzden onu <strong>taşma deposu</strong>olarak kullanabilirsin!
+
```

Comment:
Translation was fine. Why to change it though? 
**Line 652:** First letter of the first word is not in caps at.
**Line 654:** Grammatical mistake by word "buyüzden". It should be "Bu yüzden" with caps on the first letter.
The word "Taşma deposu" is properiate to use for the translation of "overflow gate".